src/box/lua/info.c

```diff
@@ -423,6 +424,9 @@ lbox_info_memory_call(struct lua_State *L)
 	lua_pushinteger(L, G(L)->gc.total);
 	lua_settable(L, -3);
 
+	luaL_pushuint64(L, tuple_runtime_memory_used());
```

Comment:
Maybe, better report the amount of memory used by the [`runtime`](https://github.com/tarantool/tarantool/blob/cbc3daed35068ea5412aa1692e2bd5f7e2153f29/src/lib/core/memory.h#L46) arena? It includes both runtime tuples and fiber stacks. It's shown by [`box.info.runtime.info()`](https://github.com/tarantool/tarantool/blob/cbc3daed35068ea5412aa1692e2bd5f7e2153f29/src/box/lua/slab.cc#L240).

Reply:
`box.runtime.info().used` is not so directly connected to amount of memory occupied by tuples allocated on runtime arena. I need a half of million of `box.tuple.new({})` calls to see a change in this metric.

Both are useful IMHO. I would like to offer a metric that specifically tracks runtime tuples. A large amount of runtime tuples (or wrong refcounting) is not rare source of problems and it would be nice to spot such situations easily (and easily reproduce with small amount of data).

TBH, I looked at possibly related client's problem today and it is why I resurrected this old patch.

Reply:
"Runtime" sounds like all runtime memory, and the fact that `box.runtime.info().used` and `box.info.memory().runtime` show different values looks confusing. Maybe, make `runtime` a table?
```
runtime:
  tuple
  fiber
  ...
```

Or show the total runtime memory in `box.info.memory().runtime` and add a different function to get the details, e.g. `box.info.memory.runtime()` would return a table showing memory usage per kind?

Anyway, this needs to be discussed with @alyapunov.

Reply:
I agree on `box.info.memory().runtime.{tuple,fiber}`, but I find `box.info.memory().runtime` callable that serializes to a number quite unusual and maybe even confusing.

How about the following?

```yaml
tarantool> box.info.memory().runtime
...
- tuple: <number>
  fiber: <number>
  sum: <number>
---
```

Reply:
Updated to `box.info.memory().runtime.tuple` in the meanwhile.

Reply:
We discussed it with @alyapunov and @locker and agreed to add the `tuple` field to `box.runtime.info()`, because this call already returns information related to runtime arena. I'll adjust the patch accordingly.